sky/packages/sky/lib/src/material/scrollbar_painter.dart

```diff
@@ -0,0 +1,87 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:ui' as ui;
+
+import 'package:flutter/animation.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/widgets.dart';
+
+const double _kMinScrollbarThumbLength = 18.0;
+const double _kScrollbarThumbGirth = 6.0;
+const Duration _kScrollbarThumbFadeDuration = const Duration(milliseconds: 300);
+
+class ScrollbarPainter extends ScrollableListPainter {
+
+  double __opacity = 0.0;
+  double get _opacity => __opacity;
+  void set _opacity (double value) {
+    __opacity = value;
+    renderer?.markNeedsPaint();
+  }
+
+  int get _alpha => (_opacity * 0xFF).round();
+
+  // TODO(hansmuller): thumb color should come from ThemeData.
+  Color get thumbColor => const Color(0xFF9E9E9E);
+
+  void paintThumb(PaintingContext context, Rect thumbBounds) {
+    final Paint paint = new Paint()
+      ..color = thumbColor.withAlpha(_alpha)
+      ..style = ui.PaintingStyle.fill;
```

Comment:
I was being defensive because /builds/dev/flutter/src/Paint.dart doesn't appear to define a default value of style. 

I've removed the style init.